### ISSUE TITLE
Rooms

### ISSUE THREAD

**Issue by vgenev** [dwyl/hapi-socketio-redis-chat-example#43]
Hi, guys, I am wondering what would be the best practice to add rooms to this chat. And how to handle the room from the client to the handler? 
Thank you.

**Comment by edwardcodes** [comment]
Good question @vgenev rooms (or "channels") would be quite straightforward to add. 
Adding *permissions* (*i.e: restricting who can read the different rooms) is a bit more involved, but also doable. The UI is the *interesting* part. If you have time to build something, let us know.

**Comment by nelsonic** [comment]
Yeah, we did not *expand* on this example because we were trying to keep it *simple* for *learning purposes*. We could turn this into something a bit more useful with Rooms, Permissions, Mentions, Markdown Parsing, Link following etc.
